po/de.po

```diff
@@ -1382,31 +1382,31 @@ msgstr ""
 "\n"
 "Weitere Details in tracepath(8).\n"
 
-#: tracepath.c:459 tracepath.c:464
+#: tracepath.c:457 tracepath.c:462
 msgid "Only one -4 or -6 option may be specified"
 msgstr "Es darf nur eine Option -4 oder -6 angegeben werden"
 
-#: tracepath.c:606
+#: tracepath.c:604
 #, c-format
 msgid "%2d:  no reply\n"
 msgstr "%2d:  keine Antwort\n"
 
-#: tracepath.c:613
+#: tracepath.c:611
 #, c-format
 msgid "     Resume: pmtu %d "
 msgstr ""
 
-#: tracepath.c:615
+#: tracepath.c:613
 #, c-format
 msgid "hops %d "
 msgstr ""
 
-#: tracepath.c:617
+#: tracepath.c:615
 #, c-format
 msgid "back %d "
 msgstr ""
 
-#: tracepath.c:622
+#: tracepath.c:620
 #, c-format
 msgid "pktlen must be within: %d < value <= %d"
 msgstr ""
```

Comment:
I think the best here would be `pktlen muss innerhalb von %d < Wert <= %d liegen`.

Or alternatively, if that doesn't work: `pktlen muss innerhalb der Grenzen liegen: %d < value <= %d` (`pktlen must be within the limits: %d < value <= %d`) or `pktlen muss innerhalb der Werte liegen: %d < Wert <= %d`